test/integration/cli/ns/extends.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
Please undo this change. 

The test is meant to verify that rules pulled in via `plugin:@ns/bpmnlint-plugin-test/recommended` are used without later activation (in the "rules" block).

Reply:
But why is it using `test2` if we aren't extending from it? I may be misunderstanding something here...

Reply:
OK got it.